ghga_event_schemas/pydantic_.py

```diff
@@ -31,6 +31,9 @@ class MetadataSubmissionFiles(BaseModel):
     file_id: str = Field(
         ..., description="The public ID of the file as present in the metadata catalog."
     )
+    file_name: str = Field(
+        ..., description="Name of the file.", example="treatment_R1.fastq.gz"
```

Comment:
Should we make it more clear, where that file name came from, since this is not the file name the file has on our system?
e.g. "The name of the file as it was submitted"

Reply:
Gerne